bluetooth.js

```diff
@@ -100,7 +100,7 @@ var BluetoothDevice = class {
     }
 
     update(iter) {
-        this.name = this._model.get_value(iter, GnomeBluetooth.Column.NAME);
+        this.name = this._model.get_value(iter, GnomeBluetooth.Column.ALIAS || GnomeBluetooth.Column.NAME);
```

Comment:
I believe it won't work.
`GnomeBluetooth.Column.ALIAS` is constant and always have value, so result will always be the same.

Shouldn't it be something like this?
```
this.name = this._model.get_value(iter, GnomeBluetooth.Column.ALIAS) || this._model.get_value(iter, GnomeBluetooth.Column.NAME);
```

Reply:
Thanks. Fixed.